pkgs/servers/monitoring/prometheus/apcupsd-exporter.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "apcupsd-exporter";
-  version = "0.2.0";
+  version = "0.3.0";
 
   src = fetchFromGitHub {
     owner = "mdlayher";
     repo = "apcupsd_exporter";
     rev = "v${version}";
-    sha256 = "0gjj23qdjs7rqimq95rbfw43m4l6g73j840svxjlmpd1vzzz2v2q";
+    sha256 = "sha256-c0LsUqpJbmWQmbmSGdEy7Bbk20my6iWNLeqtU5BjYlw=";
   };
 
-  vendorSha256 = "09x8y8pmgfn897hvnk122ry460y12b8a7y5fafri5wn9vxab9r82";
+  vendorSha256 = "sha256-bvLwHLviIAGmxYY1O0wFDWAMginEUklicrbjIbbPuUw=";
 
   doCheck = false;
```

Comment:
Any particular reason we're not running the tests? If so, could we annotate that information?

Reply:
https://github.com/NixOS/nixpkgs/pull/94632

Looks like this was done en-masse some time ago, before I touched this package. I'll try removing it.

Reply:
Looks good, removed:

```
running tests
=== RUN   TestUPSCollector
=== RUN   TestUPSCollector/empty
=== RUN   TestUPSCollector/full
--- PASS: TestUPSCollector (0.00s)
    --- PASS: TestUPSCollector/empty (0.00s)
    --- PASS: TestUPSCollector/full (0.00s)
PASS
ok      github.com/mdlayher/apcupsd_exporter    0.005s

```